content/development-update/weekly-development-report-as-of-2025-03-21.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Weekly development report as of 2025-03-21
+tags:
+  - Development
+  - Statistics
+  - Weekly development updates
+url: ""
+image: https://ucarecdn.com/cc3bd524-c357-4861-ac38-6849253d953b/
+image_text: Weekly development report as of 2025-03-21
+---
+
+### ECOSYSTEM UPDATES
+
+_The information in this section is based on publicly available sources and is provided for informational purposes only. It does not constitute an endorsement or recommendation of any project, product, or service mentioned._
+
+Cardano’s ecosystem continues to progress, with 1,989 projects currently building on the network. The number of delegated wallets has increased to 1.328 million, showing a 0.07% growth in staking participation since last week. The issuance of native tokens has reached 10.72 million, with 210,662 token policies, reflecting a 0.14% increase from the previous week.
+
+Smart contract adoption remains steady, with 129,374 Plutus scripts  and 5,691 Aiken scripts now deployed. Transaction activity continues to rise, reaching 107.69 million transactions, marking a 0.16% increase since last week. Governance participation also remains active, with 1,217 DReps, including 923 active DReps, contributing to decentralized decision-making.
```

Comment:
```suggestion
Smart contract adoption remains steady, with 129,374 Plutus scripts and 5,691 Aiken scripts now deployed. Transaction activity continues to rise, reaching 107.69 million transactions, marking a 0.16% increase since last week. Governance participation also remains active, with 1,217 DReps, including 923 active DReps, contributing to decentralized decision-making.
```